docs/architecture.md

```
@@ -183,6 +192,84 @@ Each command prepares a fully self-contained directory that can be zipped for st
+    processedLength: number
+    wasModified: boolean
+    provider: string
+    processingTime?: number
+  }
+}
+```
+
+### 7.5.2 Current Providers
+
+* **OpenAI Provider**: Translates text to Russian using OpenAI's GPT-3.5-turbo model
+  - Requires API key stored in extension storage or environment variable
```

markdownlint-cli2 (0.18.1)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)